client/src/main/scala/client/SampleClient.scala

```diff
@@ -0,0 +1,19 @@
+package client
+
+import shared.Shared
+
+import scala.scalajs.js
+import scala.scalajs.js.Dynamic._
+
+object SampleClient extends js.JSApp {
```

Comment:
<script type="text/javascript">
            $('#submit').click(function(e) {
                var report = $("#report");
                report.empty();
                var solution = $("#solution").val();

                jsRoutes.controllers.TaskSolver.postSolutionAjax(solution).ajax({
                    success: function(data){
                        report.html(data);
                    }, error: function(data) {
                        console.error(data);
                    }
                });
                return false;
            });
    </script>
Not really, but I'll migrate out js code today